app/views/subjects/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+= form_for @subject do |f|
+  .field
+    = f.label :name
+    = f.text_field :name
+  .field
+    = f.label :desc
+    = f.text_area :desc
+  .actions
+    = f.submit
+  - if @subject.errors.any?
+    #error_explaination
+    %h2= pluralize(@subject.errors.count, 'error')
```

Comment:
Also, not sure - was just converting from erb to haml (I took this code from posts and modified it for subjects and converted to haml). This is the piece of code I based it on.
` <% if @post.errors.any? %>
    <div id="error_explanation">
      <h2><%= pluralize(@post.errors.count, "error") %> prohibited this post from being saved:</h2>

      <ul>
      <% @post.errors.full_messages.each do |message| %>
        <li><%= message %></li>
      <% end %>
      </ul>
    </div>
  <% end %>`